app/models/presenters/tube_presenter.rb

```diff
@@ -46,5 +46,19 @@ def sequencescape_submission
       yield s if block_given?
       s
     end
+
+    def child_plates
+      labware.child_plates.tap do |child_plates|
+        yield child_plates if block_given? && child_plates.present?
+      end
+    end
+
+    alias child_assets child_plates
```

Comment:
What's the alias for? I would have thought that assets would normally refer to both plates and tubes